src/core/parser/csv_parser/csv_parser.cpp

```diff
@@ -110,12 +123,26 @@ std::string CSVParser::GetUnparsedLine(unsigned long long const line_index) {
 }
 
 std::vector<std::string> CSVParser::ParseString(std::string const& s) const {
+    std::string t;
+    for (size_t index = 0; index < s.size(); index++) {
```

Comment:
```suggestion
    for (std::size_t index = 0; index < s.size(); index++) {
```
nit: `std::size_t` is preferred, since `size_t` is not guaranteed to be defined by the standard

Reply:
nit: add `#include <cstddef>`